src/tseda/vpages/stats.py

```diff
@@ -218,11 +216,28 @@ def __panel__(self):
         windows = make_windows(self.window_size, tsm.ts.sequence_length)
         comparisons = eval_comparisons(self.comparisons.value)
 
-        sample_sets_dictionary = self.datastore.individuals_table.sample_sets()
-        sample_sets_ids = list(sample_sets_dictionary.keys())
-        if len(sample_sets_ids) < 2:
+        selected_sample_sets = self.datastore.individuals_table.sample_sets()
+        selected_sample_sets_ids = list(selected_sample_sets.keys())
+        if len(selected_sample_sets_ids) < 2:
             return self.sample_select_warning
-        sample_sets_individuals = list(sample_sets_dictionary.values())
+        elif self.comparisons.value == []:
+            return pn.pane.Markdown(
+                "**Select which sample sets to compare to see this plot.**"
+            )
+        all_sample_sets = self.datastore.individuals_table.sample_sets(
+            only_selected=False
+        )
+        all_sample_sets_sorted = {
+            key: all_sample_sets[key] for key in sorted(all_sample_sets)
+        }
```

Comment:
sample_sets have to be sorted (as sample sets 0 and 1 come in reverse order from sample_sets function for some reason.

Reply:
I guess that has to do with the fact that sample_sets returns a dictionary, where by default keys are unsorted. One could use an OrderedDict to avoid this issue.